.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+
+[*.php]
+indent_style = tab
+indent_size = 8
```

Comment:
https://www.mediawiki.org/wiki/Manual:Coding_conventions#Tab_size suggests 4 spaces. I ended merging this patch and editing it myself, see 8ae8c2af7a323d4ee094bd1db14f2417720c3fde. Feel free to upload another patch if you disagree with what I did. The easiest solution is to remove controversial lines.